### DISCUSSION TITLE
Domain name change causes app styling change

### DISCUSSION THREAD

**Discussion by Yiwen-Gao** [vercel/community#1598] [Help] [answered]
### Summary

The default domain assigned to my deployment was `<repo_name>.vercel.app`, but I wanted it to be something that was more specific. When I rename it to `mahkana.vercel.app`, it changed the styling of the buttons and the font completely. 

![image](https://user-images.githubusercontent.com/32348009/220529778-405d410b-95c8-4729-a44f-fc549c6e3628.png)

But when I change it to other names e.g. `mahkanaa.vercel.app`, the styling stays the same. 
![image](https://user-images.githubusercontent.com/32348009/220529875-15471f3a-138a-4360-9504-c4bde8530a82.png)

Why does this happen? What can I do to fix it?

**Comment by amyegan** [comment]
Hi @Yiwen-Gao! I wasn't able to replicate the problem in my own test app. Can you try adding the domain to your Project again so I can take a closer look at what is going wrong?

**Reply by Yiwen-Gao** [reply]
Actually, I changed it back, opened it in an incognito window, and the styling was correct! I think the styles aren't getting updated even when I do a hard refresh on my browser, so I probably have to clear my cookies or invalidate the cache somehow. Thanks for looking into this though!

**Comment by Yiwen-Gao** [comment, marked as answer]
It seems like the issue was because the styles were getting cached and weren't updated even when I do a hard refresh.
